node/node.go

```diff
@@ -1161,6 +1161,12 @@ func (node *AlgorandFullNode) VotingKeys(votingRound, keysRound basics.Round) []
 		}
 		participations = append(participations, part)
 		matchingAccountsKeys[part.Address()] = true
+
+		// This is usually a no-op, but the first time it will update the DB.
+		err := node.participationRegistry.Register(part.ParticipationID(), keysRound)
+		if err != nil {
+			node.log.Error("Failed to register participation key (%s) with participation registry.", part.ParticipationID())
```

Comment:
`Error` -> `Warnf`.
Errors are meant for critical functional issues with the node. The node would continue to work just fine even if this call fails.